explorer/client/cypress/e2e/e2e.cy.ts

```diff
@@ -0,0 +1,284 @@
+        });
+
+        it('can be reached through URL', async () => {
+            await page.goto(`${BASE_URL}/transactions/${successID}`);
+            const value = await cssInteract(page)
+                .with('#transactionID')
+                .get.textContent();
+            expect(value.trim()).toBe(successID);
+        });
+        it('correctly renders days and hours', async () => {
+            await page.goto(`${BASE_URL}/transactions/${successID}`);
+            const value = await cssInteract(page)
+                .with('#timestamp')
+                .get.textContent();
+            expect(value.trim()).toBe(
+                '17 days 1 hour ago (15 Dec 2024 00:00:00 UTC)'
+            );
+        });
+        it('correctly renders a time on the cusp of a year', async () => {
+            const otherID = 'GHTP9gcFmF5KTspnz3KxXjvSH8Bx0jv68KFhdqfpdK8=';
+            await page.goto(`${BASE_URL}/transactions/${otherID}`);
+            const value = await cssInteract(page)
+                .with('#timestamp')
+                .get.textContent();
+            expect(value.trim()).toBe(
+                '1 min 3 secs ago (01 Jan 2025 01:12:07 UTC)'
+            );
+        });
+        it('correctly renders a time diff of less than 1 sec', async () => {
+            const otherID = 'XHTP9gcFmF5KTspnz3KxXjvSH8Bx0jv68KFhdqfpdK8=';
+            await page.goto(`${BASE_URL}/transactions/${otherID}`);
+            const value = await cssInteract(page)
+                .with('#timestamp')
+                .get.textContent();
+            expect(value.trim()).toBe('< 1 sec ago (01 Jan 2025 01:13:09 UTC)');
+        });
+    });*/
+
+    describe('Owned Objects have links that enable', () => {
+        it('going from address to object and back', () => {
+            cy.visit('/addresses/receiverAddress');
+            cy.get(nftObject(1)).click();
+            cy.get('#objectID').contains('player1');
+            cy.get(ownerButton).click();
+            cy.get('#addressID').contains('receiverAddress');
+        });
+
+        it('going from object to child object and back', () => {
+            cy.visit('/objects/player2');
+            cy.get(nftObject(1)).click();
+            cy.get('#objectID').contains('Image1');
+            cy.get(ownerButton).click();
+            cy.get('#objectID').contains('player2');
+        });
+
+        it('going from parent to broken image object and back', () => {
+            const parentValue = 'ObjectWBrokenChild';
+            cy.visit(`/objects/${parentValue}`);
+            cy.get(nftObject(1)).click();
+            cy.get('#noImage');
+            cy.get(ownerButton).click();
+            cy.get('#loadedImage');
+        });
+    });
+
+    describe('PaginationWrapper has buttons', () => {
+        it('to go to the next page', () => {
+            const address = 'ownsAllAddress';
+            cy.visit(`/addresses/${address}`);
+            cy.get('[data-testid="nextBtn"]').filter(':visible').click();
```

Comment:
Following on from the problem encountered in the PR https://github.com/MystenLabs/sui/pull/3924, this code will need to be updated to handle the case of when there are multiple paginations on the same page. In the mentioned PR there was a problem where Puppeteer was clicking the first buttons it saw, which were in the Owned Coins section of the page, rather than the buttons we wanted it to click, which were in the Owned NFTs section of the page.

Hence, could we have something like `[data-testid="NFTSection"] [data-testid="nextBtn"]` so Cypress clicks the correct button? This update will need to be applied to all tests of the pagination buttons.

Reply:
Thanks for calling this out! Updated the tests to lock it to a section

Reply:
Thank you for looking into this. The updated tests lock to the section Owned Objects. Both Owned Coins and Owned NFTs are children of Owned Objects. The framework was being confused between these two sections. Hence, the tests will need to be locked specifically to Owned NFTs.